samcli/local/apigw/local_apigw_service.py

```diff
@@ -328,9 +340,7 @@ def _construct_event(flask_request, port, binary_types):
 
         request_data = flask_request.get_data()
 
-        request_mimetype = flask_request.mimetype
```

Comment:
Revert this back.

Reply:
I was changing it to get under the style check of R:318, 4: Too many local variables. It is also only used once.